internal/resources/akscluster/resource_akscluster_acc_test.go

```diff
@@ -498,13 +501,14 @@ func mockCluster(w ...clusterWither) *aksmodel.VmwareTanzuManageV1alpha1AksClust
 			Config: &aksmodel.VmwareTanzuManageV1alpha1AksclusterClusterConfig{
 				Location: "eastus",
 				NetworkConfig: &aksmodel.VmwareTanzuManageV1alpha1AksclusterNetworkConfig{
-					DNSPrefix: "dns-tf-test",
+					NetworkPlugin: "azure",
+					DNSPrefix:     "dns-tf-test",
 				},
 				StorageConfig: &aksmodel.VmwareTanzuManageV1alpha1AksclusterStorageConfig{
 					EnableDiskCsiDriver: false,
 					EnableFileCsiDriver: false,
 				},
-				Version: "1.25.11",
+				Version: "1.26.6",
```

Comment:
is it intentional change?

Reply:
I do see some test config changes, what is the need of those changes? It will good to capture it in the PR description

Reply:
Yes, as per the EKS credential created these versions are supported and the config changes are also according to that.

Reply:
Ok Lets get this reviewed from AKS team too.

@gargi @asutoshpalai - Can one of you please review this change?